credential/reclaim/v0.1.0/schema.json

```diff
@@ -0,0 +1,133 @@
+                                        "type": "string"
+                                    },
+                                    "identifier": {
+                                        "title": "Identifier of request",
+                                        "description": "Identifier of request",
+                                        "type": "string"
+                                    },
+                                    "epoch": {
+                                        "title": "Epoch of request",
+                                        "description": "Epoch of request",
+                                        "type": "string"
+                                    }
+                                }
+                            },
+                            "signature": {
+                                "title": "Signature",
+                                "description": "The signature of verification credential",
+                                "type": "array",
+                                "items": {
+                                    "type": "string"
+                                }
+                            },
+                            "witnesses": {
+                                "title": "witnesses",
+                                "description": "Witness",
+                                "type": "array",
+                                "items": {
+                                    "type": "object",
+                                    "properties": {
+                                        "id": {
+                                            "title": "ID",
+                                            "description": "ID",
+                                            "type": "string"
+                                        },
+                                        "url": {
+                                            "title": "URL",
+                                            "description": "URL",
+                                            "type": "string"
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    }
+                },
+                "sessionId": {
+                    "title": "Session Id",
+                    "description": "Session Id of process",
+                    "type": "string"
+                },
+                "reclaimProviderId": {
+                    "title": "Reclaim provider ID",
+                    "description": "The schema ID that added in the project",
+                    "type": "string"
+                },
+                "reclaimProviderLabel": {
+                    "title": "Reclaim provider Label",
+                    "description": "The schema label that added in the project",
+                    "type": "string"
+                },
+                "did": {
```

Comment:
In the W3C Verifiable Credential specification, the subject's DID is referred in an `id` property, we should use this rather than `did`

Reply:
Done